src/analysis/special_functions/trigonometric.lean

```diff
@@ -746,10 +746,10 @@ calc cos y = cos x * cos (y - x) - sin x * sin (y - x) :
   exact sub_le_self _ (mul_nonneg (sin_nonneg_of_nonneg_of_le_pi hx₁ (by linarith))
     (sin_nonneg_of_nonneg_of_le_pi (by linarith) (by linarith)))
 
-lemma cos_lt_cos_of_nonneg_of_le_pi {x y : ℝ} (hx₁ : 0 ≤ x) (hx₂ : x ≤ π)
+lemma cos_lt_cos_of_nonneg_of_le_pi {x y : ℝ} (hx₁ : 0 ≤ x)
```

Comment:
You can also remove 0<= y, right? Because x<y. I don't know if that's a good idea though!